Thanks!

Indeed, I try to build `julia` (which used to build correctly).

Now, the error is as follows:

```
In file included from /nix/store/z5cz1s68y8ixpcqk3nzsncc42jlslg6v-libunwind-osx-10.11.6/include/libunwind.h:23:
/nix/store/z5cz1s68y8ixpcqk3nzsncc42jlslg6v-libunwind-osx-10.11.6/include/libunwind-x86_64.h:127:10: fatal error: 'libunwind-common.h' file not found
#include "libunwind-common.h"
```

So `libunwind.h` is there, but some files it relies on are not.

--- a/pkgs/os-specific/darwin/apple-source-releases/libunwind/default.nix
+++ b/pkgs/os-specific/darwin/apple-source-releases/libunwind/default.nix
@@ -1,15 +1,16 @@
 { stdenv, appleDerivation, dyld, osx_private_sdk }:
 
 appleDerivation {
-  phases = [ "unpackPhase" "installPhase" ];
+  buildPhase = ":";
 
+  # install headers only
   installPhase = ''
     mkdir -p $out/lib
-    cp -R include $out/include
+    make install-data-am
   '';
 
   meta = with stdenv.lib; {
-    maintainers = with maintainers; [ copumpkin ];
+    maintainers = with maintainers; [ copumpkin lnl7 ];
     platforms   = platforms.darwin;
     license     = licenses.apsl20;
   };

